script/testing/junit/src/TestUtility.java

```diff
@@ -127,4 +137,43 @@ public void setValues(PreparedStatement pstmt, int[] values) throws SQLException
             pstmt.setInt(col++, (int) values[i]);
         }
     }
+
+    /**
+     * Get all sql query statement start line numbers
+     * @param input test file
+     * @return list of integers that contains start line numbers
+     * @throws IOException
+     */
+    public static List<Integer> getQueryLineNum(File input) throws IOException {
+        BufferedReader bf = new BufferedReader(new FileReader(input));
+        List<Integer> res = new ArrayList<>();
+        String line;
+        int counter = 0;
+        while (null != (line = bf.readLine())){
+            counter++;
+            if(line.startsWith("query") || line.startsWith("statement")){
+                res.add(counter);
+            }
+        }
+        return res;
+    }
```

Comment:
Is this function only needed by `TraceFileTest`? If yes, then put it inside that class.